addon/mixin.js

```diff
@@ -83,7 +83,7 @@ export default Ember.Mixin.create({
     var content = this.get('content');
 
     keys(buffer).forEach(function(key) {
-      if (isArray(onlyTheseKeys) && !onlyTheseKeys.contains(key)) {
+      if (isArray(onlyTheseKeys) && !Ember.A(onlyTheseKeys).contains(key)) {
```

Comment:
Had to remove these two uses of prototype extensions because of `ember-disable-prototype-extensions`.

I can remove the package if you'd rather that be done separately.